README.md

```diff
@@ -241,7 +241,7 @@ export ONEVIEWSDK_PROXY='<proxy_host>:<proxy_port>'
 
 The OneView Python SDK supports the API endpoints for HPE OneView 4.10, 4.20, 5.00, 5.20, 5.30.
 
-The current `default` HPE OneView version used by the Python SDK is `4.10`, API `800`.
+The current `default` HPE OneView version used by the Python SDK will pick the version used by OneView appliance provided by customer.
```

Comment:
The current `default` HPE OneView version used by the Python SDK will pick the OneView appliance version.

Reply:
done